### ISSUE TITLE
nxtgn provider not picking up results AND login issues

### ISSUE THREAD

**Issue by Symbiot78** [SiCKRAGETV/sickrage-issues#942]
Branch - Master
Commit hash -  (3b399f16126c2566028312ecedcbb0a0d47d3dfa)  
Your operating system and python version - Open Media Vault - 2.7
What you did - Enabled nxtgn provider
What happened - it doesn't find anything, and it sporadically give me prompts on nxtgn.org about failed login attempts. Now I have TRIPLE checked username & password..  It worked before the last 3 commits but now it doesn't. I know this isn't the easiest of issues and with the username/password issue it's easy to blame me for mistyping but I guarantee that that is not the issue.
The log reports  'The Data returned from NextGen do not contains any torrent' but when I copy paste the searchURL it returns results.
'NextGen search page URL: https://nxtgn.org/browse.php?search=Criminal.Minds S10E17 &cat=0&incldead=0&modes=&c7=1&c24=1&c17=1&c22=1&c42=1&c4$'

When I do a manual search of a 'missed show' in 'coming episodes' I don't see any warnings on nxtgn.org.. I think it might only be when running daily search or one of the other searches.. but I cannot see anything in the log. And I've tried manually initiating the daily & backlog and that also doesn't warn me on nxtgn.org..

I hope something in the log makes sense to you.. I can't see it.. and I can't see any additional debugging eventhough I enabled debud in config.ini... AND checked that it was still enabled after shutdown / start of SickRage.

What you expected - that it works :-)

Link to a copy/paste of your logfile with clear debug info of the error on ;;
link to log file
http://pastebin.com/7rSxAB8B

**Comment by abeloin** [comment]
In Search Providers -> Configure Custom Newznab Providers, select your provider and make sure 5030 and 5040 are in Newznab search categories on the right side

**Comment by Symbiot78** [comment]
Hi Abeloin..
It's a torrent provider, and it's not a custom torrent provider. It's just a normal provider added by Sickbeard team. 

what is 5030 & 5040  ?

**Comment by Symbiot78** [comment]
I have since tried reinstalling sickbeard and can confirm that the issue persists.

**Comment by abeloin** [comment]
Sorry, I didn't see in the list so I assume it was a newznab provider. NextGen

Maybe the web site changed? Can you provide a full html copy of that search url via dropbox or any other file locker?

**Comment by Symbiot78** [comment]
Hi

you can download the log file here it has more than 1 of the urls:

https://www.dropbox.com/s/7xvha6vg3rh35tx/sickrage.log?dl=0

I tried taking one of the URLS it is using to search and paste it into my browser and it does indeed find results.. but according to the log.. it doesn't find anything. 
and I am still seeing username/password warnings from nxtgn.org as soon as I activate the nxtgn plugin.

**Comment by Symbiot78** [comment]
Hi

just to add to this issue.. I checked the config.ini file and can confirm that the username/password is indeed correct.

**Comment by abeloin** [comment]
There's no warning in the log provided.

What is the title tag value in the HTML after logging? Currently it's looking for: `<title>NextGen - Login</title>` to know if you are logged.

Also we don't have access to this site this is why I asked for a dump of an html page with results once logged as this plugin scrap data from the web pages directly.

**Comment by Symbiot78** [comment]
Hi
If you can tell me how to create the dump you want I'll be happy to do it.I don't think the plugin ever gets access to the site.

Sent from Outlook




On Sat, Mar 14, 2015 at 7:11 AM -0700, "Alexandre Beloin" <notifications@github.com> wrote:
There's no warning in the log provided.

What is the title tag value in the HTML after logging? Currently it's looking for: `<title>NextGen - Login</title>` to know if you are logged.

Also we don't have access to this site this is why I asked for a dump of an html page with results once logged as this plugin scrap data from the web pages directly.

---

**Comment by abeloin** [comment]
Login in the site with any broswer, go to: https://nxtgn.org/browse.php and press CTRL+S and save it. Send the resulting page

**Comment by Symbiot78** [comment]
that' swhat i thought.. just wanted to make sure.

here's a dropbox link

https://www.dropbox.com/s/7cw0szj4zn266by/nxtgn.rar?dl=0

it contains the html etc. of a search for an episode I have on my wanted in SB but which isn't giving me any results :-)

**Comment by abeloin** [comment]
The html look ok in sickrage.

> I am still seeing username/password warnings from nxtgn.org as soon as I activate the nxtgn plugin.
Can you show me these warning?

**Comment by Symbiot78** [comment]
Hi

it's in Danish.. 
All it says is:

" A login attempt for this account has just failed from IP xx.xx.xx.xx (my IP)
It was tried with the following password : Pas..ord

so it's showing part of my password (I changed it above)

I suspected Sickbeard and tried changing my password for nxtgn in Sickbeard so instead of using my real password in the SB nxtgn plugin I used another one: Cheesestake

and sure enough.. next time the nxtgn system alerted me about a failed login attempt the message read:

" A login attempt for this account has just failed from IP xx.xx.xx.xx (my IP)
It was tried with the following password : Che....teak

So it's obviously coming from sickbeard.. and something is going wrong.. I just cannot figure out what.

I tried entering my real password again but no changes.. still fails.

**Comment by abeloin** [comment]
Do you the 2 factor authentication enable on the site?

**Comment by Symbiot78** [comment]
Hi ..

No..

I've just done a complete reinstall. Making sure everything SickRage was removed.
I triple checked any residual Sickrage files rebooted and reinstalled Sickrage master branch. 

I shifted all pointers (folders) to a different location and redid everything... including the nxtgn plugin. 

For some reason.. it now connects to nxtgn properly. Meaning it works.

Same username, same password.

Only issue I have right now is that using the Download station torrent downloader doens't work. But I can use a black hole folder for that.


Now what the hell happened I don't know. But something was off before..  and now.. it isn't. Whether is a bug or because my original SickRage installation was old and had been updated many times And perhaps had left something behind during my last attempt at a reinstall I don't know.. but this time it apparently worked.

I am unsure what to do now... just leave it at that or dig some more into the issue...
